l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -51,35 +54,45 @@ def ProvinceByCode(self, cr, uid, provinceCode, context=None):
     def check_partner_base_data(
         self, cr, uid, partner_id, DatiAnagrafici, context=None
     ):
+        if context is None:
+            context = {}
         partner = self.pool['res.partner'].browse(
             cr, uid, partner_id, context=context)
         if (
             DatiAnagrafici.Anagrafica.Denominazione
             and partner.name != DatiAnagrafici.Anagrafica.Denominazione
         ):
-            raise orm.except_orm(
-                _('Error!'),
-                _('XML content \'%s\' different from system data \'%s\'. '
-                  'Please fix your data')
-                % (DatiAnagrafici.Anagrafica.Denominazione, partner.name))
+            if context.get('inconsistencies'):
+                context['inconsistencies'] += '\n'
+            context['inconsistencies'] += (
+                _(
+                    "DatiAnagrafici.Anagrafica.Denominazione contains %s."
+                    " Your System contains %s"
+                )
+                % (DatiAnagrafici.Anagrafica.Denominazione, partner.name)
+            )
         if (
             DatiAnagrafici.Anagrafica.Nome
             and partner.firstname != DatiAnagrafici.Anagrafica.Nome
         ):
-            raise orm.except_orm(
-                _('Error!'),
+            if context.get('inconsistencies'):
+                context['inconsistencies'] += '\n'
+            context['inconsistencies'] += (
                 _('XML content \'%s\' different from system data \'%s\'. '
                   'Please fix your data')
```

Comment:
Modificherei questo messaggio ed il successivo in modo che sia coerente con quello sopra. Deve essere chiaro che i dati sono stati importati ma sono state trovate incongruenze